sympy/logic/boolalg.py

```diff
@@ -914,6 +914,16 @@ def to_nnf(self, simplify=True):
         a, b, c = self.args
         return And._to_nnf(Or(~a, b), Or(a, c), simplify=simplify)
 
+    def _eval_derivative(self, x):
+        return self.func(self.args[0], *[a.diff(x) for a in self.args[1:]])
+
+    # the diff method below is copied from Expr class
```

Comment:
Well, the short story is that `_eval_derivative` was not available in ITE so I thought this would be the least invasive change. However, copy-and-pasting code is obviously not a great solution, so I am open to suggestions.